app.go

```diff
@@ -244,24 +246,31 @@ func pushImage(image string) {
 	execBuild("./", []string{"docker", "push", image})
 }
 
-func buildImage(image string, handler string, functionName string, language string, nocache bool) {
+func buildImage(image string, handler string, functionName string, language string, nocache bool, squash bool) {
 
 	switch language {
 	case "node", "python":
 		tempPath := createBuildTemplate(functionName, handler, language)
 
 		fmt.Printf("Building: %s with Docker. Please wait..\n", image)
 
-		cacheFlag := ""
+		//lets create a buffer for any build flags
+		var flagBuf bytes.Buffer
```

Comment:
It could.  What I've read though suggests that using byte buffer is better performance wise when there is any kind of variability in length - strings are immutable.  Less of a consideration with 3 functions and four concats per function, granted, but I'd hope users would be using the CLI for batching many more functions.